main/adapters/database/database.go

```diff
@@ -369,12 +362,6 @@ func (dm *DatabaseManager) isRecoverable(err error) bool {
 			case "55P03", "53300", "53400": // lock_not_available, too_many_connections, configuration_limit_exceeded
 				time.Sleep(10 * time.Millisecond)
 				return true
-			case "42P01": // undefined_table
```

Comment:
is this error not anymore needed?

Reply:
no, this error is not expected anymore since the migration scripts take care of the table creation upon startup

Reply:
Let us keep this object under the loop for the coming refactoring.

Reply:
of course, it makes sense.